tests/testthat/test_statistical.R

```diff
@@ -187,6 +189,21 @@ test_that("correlation", {
     expect_true(is.numeric(correlation_s_sp_mat$estimate))
     expect_true(is.matrix(correlation_s_sp_mat$estimate))
     
+    ## ggm
+    expect_true(all(correlation_g_mat$estimate -
+                        GeneNet::ggm.estimate.pcor(t(mat_test[1:5, ]), method = "static")[seq_len(dim(mat_test[1:5, ])[1]), seq_len(dim(mat_test[1:5, ])[1])]== 0))
+    expect_equal(sum(correlation_g_mat$estimate), 5.421633, tolerance = 1e-06)
+    expect_equal(sum(correlation_g_mat$p), 2.889295, tolerance = 1e-06)
+    expect_equal(rownames(correlation_g_mat$estimate), 
+                 colnames(correlation_g_mat$estimate))
+    expect_equal(rownames(correlation_g_mat$estimate), rownames(mat_test[1:5, ]))
+    expect_equal(ncol(correlation_g_mat$estimate), 
+                 nrow(correlation_g_mat$estimate))
+    expect_equal(nrow(correlation_g_mat$estimate), nrow(mat_test[1:5, ]))
+    expect_true(is.numeric(correlation_g_mat$estimate))
+    expect_true(is.matrix(correlation_g_mat$estimate))
+    expect_true(max(correlation_g_mat$estimate) <= 1)
+    expect_true(min(correlation_g_mat$estimate) >= -1)
```

Comment:
could you also add a test where the `mat_test` object contains some `NA` values (write to `mat_test_NA`) and check if the `correlation` with `"ggm"` performs as expected?

Reply:
I will change that depending on whether we want to keep the `na.omit(x) `or not